css/style.css

```diff
@@ -6,6 +6,63 @@
 #fp-nav ul li .fp-tooltip {
     color: black;
 }
+
+/* left and right panels ---------------------------------------------------------- */
+.container {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    height: 99%;
+    width: 99%;

```

Comment:
By curiosity, why 99% ?

Reply:
because fullpage.js has a 1% left margin, this compensate